dataprofiler/profilers/profile_builder.py

```diff
@@ -784,16 +784,15 @@ def _remove_data_labelers(self):
         """
         Helper method for removing all data labelers before saving to disk.
 
-        :return: data_labeler use for unstructured labeling
+        :return: data_labeler used for unstructured labelling
         :rtype: DataLabeler
         """
         data_labeler = None
 
         # determine if the data labeler is enabled
-        # TODO: fix to correct options when unstructured options avail
         use_data_labeler = True
-        if self.options and isinstance(self.options, ProfilerOptions):
-            data_labeler_options = self.options.structured_options.data_labeler
+        if self.options and isinstance(self.options, UnstructuredOptions):
+            data_labeler_options = self.options.data_labeler
             use_data_labeler = data_labeler_options.is_enabled
 
         # remove the data labeler from options
```

Comment:
I think it would rarely ever fail, but its possible that data_labeler_options would never be set, but still be called on line 800, which could cause an error. I'm not going to request this change, but just pointing it out if you want to make it more robust

Reply:
Thats a good point, it should be fine since self.options gets set even if one isn't provided, so unless the user messes around with it and makes it something else, we should be fine. Truth be told I'm not sure why we even check the instance here, it seems it should be somewhat obvious. But you make a good point.